comid-extensions.cddl

```diff
@@ -2,58 +2,18 @@
 ; Measurement Values map extensions
 ;
 $$measurement-values-map-extension //= (
-  comid.spdm-indirect => spdm-indirect-map
+  &(comid.spdm-indirect: 12) => spdm-indirect-map
 )
 spdm-indirect-map = {
-  spdm.index => [ + uint ]
+  &(spdm.index: 0) => [ + uint ]
   * $$spdm-indirect-map-extension
 }
 
 ;
 ; Flags map extensions
 ;
 $$flags-map-extension //= (
-  comid.operational-flag-runtime-meas => bool
-  comid.operational-flag-immutable-meas => bool
-  comid.operational-flag-tcb-meas => bool
+  &(runtime-meas: 6) => bool
```

Comment:
Question: Does this change make the flags-map-extension a choice between these three options (6,7,8)?

Reply:
All extensions are treated as optional since the extension is technically optional. If a ? is added it throws a syntax error. I think this is incorrect behavior personally. The human readable representation is also a goal of CDDL and even if the extension made it optional automatically, adding ? should be treated as idempotent.

Reply:
However, when I removed one of the flags from the test, the test failed. And when I added '?' to the cddl the test passed.

Reply:
Ok, that makes sense